README.md

```diff
@@ -27,31 +27,35 @@ from garminconnect import (
 )
 
 today = date.today()
+client = None
 
 
 """Login to portal using specified credentials"""
-    try:
-        client = Garmin(YOUR_EMAIL, YOUR_PASSWORD)
-    except (
-        GarminConnectConnectionError,
-        GarminConnectAuthenticationError,
-        GarminConnectTooManyRequestsError,
-    ) as err:
-        print("Error occured during Garmin Connect Client setup: %s", err)
-        return
-    except Exception:  # pylint: disable=broad-except
-        print("Unknown error occured during Garmin Connect Client setup")
-        return
-
-"""Get Full name"""
-print(client.get_full_name()
-
-"""Get Unit system"""
-print(client.get_unit_system()
-
-"""Fetch your activities data"""
-print(client.get_stats(today.isoformat())
-
-"""Fetch your logged heart rates"""
-print(client.get_heart_rates(today.isoformat())
+try:
+    client = Garmin(YOUR_EMAIL, YOUR_PASSWORD)
+except (
+    GarminConnectConnectionError,
+    GarminConnectAuthenticationError,
+    GarminConnectTooManyRequestsError,
+) as err:
+    print("Error occured during Garmin Connect Client setup: %s", err)
+    return
+except Exception:  # pylint: disable=broad-except
+    print("Unknown error occured during Garmin Connect Client setup")
+    return
+
+#"""Get Full name"""
```

Comment:
I think switching from switching from """ to a comment makes sense, but then shouldn't we drop the """?

Reply:
> My impression from what I've read is that the body fat % number is generally incorrect, but systematically so. I.e if you lose weight it should trend downwards. (I'm a recent garmin index purchaser and keen on the features in this PR!)

That is my impression too. I don't have a easy way to measure body fat% accuracy (probably go to a physicist and take a test, weight in same day before eat or drink and compare?). But to be honest, I don't care if the thing is very precise in body fat or muscle readings, but do care about weight. If other numbers give me a ballpark to work with, that's enough to my needs. Just need to know if I am going in the right direction ;)

Reply:
> I think switching from switching from """ to a comment makes sense, but then shouldn't we drop the """?

That sounds good, but let @cyberjunky tell us what he wants to do.